frontend/src/views/Layout/MainLayout.vue

```diff
@@ -105,23 +105,18 @@ export default class MainLayout extends Vue {
   userClaims: any;
 
   async mounted() {
-    // @ts-ignore
-    this.navLinks = this.$router.options.routes.find(
-      (route: any) => route.path === "/"
-    ).children;
     this.userClaims = await this.getUserClaims();
   }
 
   get routes(): any {
-    // @ts-ignore
-    return this.navLinks.filter((route: any) => {
+    return getRouteChildren().filter((route: any) => {
       return route.meta && (route.meta.activator || route.meta.menuItem);
     });
   }
 
   getMenuItems() {
```

Comment:
This whole function also seems redundant. Just use `getMenuItems` from the router directly.